.circleci/config.yml

```diff
@@ -129,19 +129,19 @@ version: 2.1
 jobs:
   build-securedrop-client:
     docker:
-      - image: circleci/python:3.5-stretch
+      - image: circleci/python:3.7-buster
     steps:
       - checkout
       - *installdeps
       - *fetchwheels
       - *clonesecuredropclient
-      - *getlatestreleasedversion
+      - *getnightlyversion
```

Comment:
We're building the latest release to make sure that we can build the current released version of each project with whatever logic is in this repo, can you expand on the motivation for this change?

Reply:
ahh you said this in the PR description. how about we tag a new version of the client?

Reply:
heyyo, I prepared this over in https://github.com/freedomofpress/securedrop-client/pull/625